components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -56,20 +64,44 @@ export class ProjectsDropdownComponent implements OnInit {
     }
 
     if (this.active) {
-      this.onSelection.emit({ detail: this.projects });
+      this.onSelection.emit(Object.values(this.selectedProjects));
       this.active = false;
     }
   }
 
   updateProjects(checked: boolean, index): void {
-    this.projects[index].enabled = checked;
+    this.updateProjectsSelected(this.projects[index], checked);
   }
 
   checkboxKeyPress(event, checked, index) {
-    this.projects[index].enabled = !checked;
+    this.updateProjectsSelected(this.projects[index], !checked);
     event.target.setAttribute('checked', !checked);
   }
 
+  updateProjectsSelected(project: Project, enabled: boolean): void {
```

Comment:
`updateSelectedProjects` reads a bit better. i might even simplify to `updateLabel`